test/TestClass.cpp

```diff
@@ -52,6 +52,25 @@ unittest(clearBuffer) {
   assertEqual(0, lines.at(1).length());
 }
 
+unittest(display_noDisplay) {
+  LiquidCrystal_Test lcd(rs, enable, d4, d5, d6, d7);
+
+  bool isDisplay = lcd.isDisplay();
+
+  // default
+  assertEqual(0, isDisplay);
```

Comment:
This could be simplified and clarified by removing the use of 0/1 for false/true:
```
assertFalse(lcd.isDisplay());
```